src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -356,12 +358,20 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
         <Box backgroundColor="backgroundFloating" width="full" style={{ backdropFilter: 'blur(24px)' }}>
           <ActionsContainer>
             <Row gap="12">
-              <FilterButton
-                isMobile={isMobile}
-                isFiltersExpanded={isFiltersExpanded}
-                onClick={() => setFiltersExpanded(!isFiltersExpanded)}
-                collectionCount={collectionNfts?.[0]?.totalCount ?? 0}
-              />
+              <TraceEvent
+                events={[Event.onClick]}
+                element={ElementName.NFT_FILTER_BUTTON}
+                name={EventName.NFT_FILTER_OPENED}
+                shouldLogImpression={!isFiltersExpanded}
+                properties={{ collection_address: contractAddress }}
```

Comment:
pls add chain_id here too. anywhere we log collection_address should include chain_id in general